openstack/limes/files/commitment_mail.tpl

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta xmlns:v="urn:schemas-microsoft-com:vml">
+        <meta xmlns:o="urn:schemas-microsoft-com:office:office">
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <style>
+            table {
+              border-collapse: collapse;
+              width: 100%;
+            }
+            
+            th, td {
+              border: 1px solid;
+              text-align: left;
+              padding: 8px;
+            }
+            </style>
+    </head>
+    <body>
+        <p><b>Dear SAP Cloud Infrastructure user</b>,</p>
+        <p>the following commitments are about to {{ .condition }} for the
```

Comment:
If condition is "be confirmed", then this will say that commitments are about to be confirmed, which is not accurate. They are already confirmed.
```suggestion
        <p>the following commitments are {{ .condition }} for the
```

And then say either "about to expire" or "confirmed and ready to use".